### DISCUSSION TITLE
Can't get classes to work in 3.0.8

### DISCUSSION THREAD

**Discussion by TisaA** [tailwindlabs/tailwindcss#6853] [Help] [answered]
Hi, I'm new to Tailwind and pretty much love it. Had a personal project going on where I used 2.2.19, took a break and now I would like to use v3.

I am trying to do a fresh install by reading through this link: https://tailwindcss.com/docs/installation 
but I can't get anything to work, there's no error message or anything like that.

My node version is 17.3, I'm using LiveServer to preview the pages.

Is there any other tutorial or anything like that might help me? Everything else I've read through seems to take from the link above which isn't working for me.

**Comment by adamwathan** [comment, marked as answer]
Hey! What do you mean by not working? Can you share your project? My first suggestion would be to make sure your `content` is configured if the classes aren't being generated:

https://tailwindcss.com/docs/content-configuration

**Reply by TisaA** [reply]
So, I've read through the content link you posted and I am unsure if that was my problem or not but the classes are now working i.e. I am now getting different styles and such after making sure the paths are configured right. So thanks for that part.

The reason I say I am unsure if that was the issue, is that I think I might've have stumbled on something else. This is the first time I've ran this: `npx tailwindcss -i ./src/input.css -o ./dist/output.css --watch` and LiveServer doesn't refresh the page on save when this is running. Not sure if that's a bug or just the way it has to be since they both rely on save but this also could have been my issue earlier.

Tailwind is working now though, so thanks for your help!
